src/com/team949/Robot.java

```diff
@@ -105,133 +107,60 @@ public void disabledPeriodic() {
+		// char startingPosition = startingPositionChooser.getSelected();
+		// String targetScoring = targetScoringChooser.getSelected();
+		// String gameData =
+		// DriverStation.getInstance().getGameSpecificMessage();
+		// autonomousCommand = autonomousSwitchLogic(startingPosition,
+		// targetScoring, gameData);
+
+		// String autoSelected = SmartDashboard.getString("Auto Selector",
+		// "Default");
+		// switch(autoSelected) {
+		// case "My Auto": autonomousCommand = new MyAutoCommand();
+		// break;
+		// case "Default Auto":
+		// default:
+		// autonomousCommand = new ExampleCommand(); break; }
 		// schedule the autonomous command (example)
 		if (autonomousCommand != null)
 			autonomousCommand.start();
 	}
-	/*private Command autonomousSwitchLogic(char startingPosition, String targetScoring, String gameData) 
-	{	
-		return new 
-		final char L = 'L';
-		final char M = 'M';
-		final char R = 'R';
-		char firstSwitch = gameData.charAt(0);
-		char scale = gameData.charAt(1);
-		
-		switch (targetScoring)
-		{
-		case "Switch":
-			if(firstSwitch == L)
-			{
-				if(startingPosition == L) 
-				{
-					return new LeftSideLeftSwitch();
-				}
-				else if(startingPosition == M) 
-				{
-					return new MiddleSideLeftSwitch();
-				}
-				else if(startingPosition == R) 
-				{
-					return new RightSideLeftSwitch();
-				}
-			}
-			else if(firstSwitch == R) 
-			{
-				if(startingPosition == L) 
-				{
-					return new LeftSideRightSwitch();
-				}
-				else if(startingPosition == M) 
-				{
-					return new MiddleSideRightSwitch();
-				}
-				else if(startingPosition == R) 
-				{
-					return new RightSideRightSwitch();
-				}
-			}
-			else 
-			{
-				System.out.println("Something went wrong ");
-				throw new UnsupportedOperationException();
-			}
-			break;
-		case "Scale":
-			if(scale == L)
-			{
-				if(startingPosition == L) 
-				{
-					return new LeftSideLeftScale();
-				}
-				else if(startingPosition == M) 
-				{
-					return new MiddleSideLeftScale();
-				}
-				else if(startingPosition == R) 
-				{
-					return new RightSideLeftScale();
-				}
-			}
-			else if(scale == R) 
-			{
-				if(startingPosition == L) 
-				{
-					return new LeftSideLeftScale();
-				}
-				else if(startingPosition == M) 
-				{
-					return new MiddleSideLeftScale();
-				}
-				else if(startingPosition == R) 
-				{
-					return new RightSideLeftScale();
-				}
-			}
-			else 
-			{
-				System.out.println("Something went wrong ");
-				throw new UnsupportedOperationException();
-			}
-			break;
-		
-		case "AutoLine":
-			if(startingPosition == L) 
-			{
-				return new LeftAutoLine();
-			}
-			else if(startingPosition == M) 
-			{
-				return new MiddleAutoLine();
-			}
-			else if(startingPosition == R) 
-			{
-				return new RightAutoLine();
-			}
-			break;
-		case "Default":
-			default:
-				return new Wait();
-		}		
-		
-		
-		return targetCommand;
-	}
 
-	/**
-	 * This function is called periodically during autonomous
+	/*
```

Comment:
Again, better to delete than to have large blocks of code sitting around commented out.  If there's a plan to uncomment this soon, go ahead and merge it.